scalableminds/webknossos
# Allow tiff export via worker for all datasets a user can see

### Steps to test:
- `yarn enable-jobs`
- create two organizations (use isDemoInstance in application.conf)
- you should be able to tiff-export on all datasets you are allowed to see, even if they belong to a different organization
- you should not be allowed to send tiff-export requests for datasets not shared with you (via team or public)

### Issues:
- fixes #5254 

------
